js/skq-global.js

```diff
@@ -1,13 +1,15 @@
 $(document).ready(function() {
 
     var body = $('html, body');
-    var windowHeight = $(window).height()
-    var scrollPos = document.body.scrollTop
-    var mobileBreak = 992
-
+    var windowHeight = $(window).height();
+    var scrollPos = document.body.scrollTop;
+    var mobileBreak = 992;
+    var logoCarousel = $(".logo-carousel");
+    var navBar = navbar;
```

Comment:
This breaks for me.

Run `bundle exec jekyll s`, open `http://localhost:4000` and look at the JS console.